.github/workflows/CI.yml

```diff
@@ -17,6 +17,8 @@ jobs:
       matrix:
         version:
           - 1
+          - 'lts'
+          - 'pre'
```

Comment:
Just to be clear 'pre' is an official release candidate and not just nightly right? I previously tested against nightly, but found it too noisy.

Reply:
Yes, it is a pretty new feature https://github.com/julia-actions/setup-julia?tab=readme-ov-file#examples